src/PatronDetail/PatronDetail.js

```diff
@@ -48,19 +48,19 @@ const PatronDetail = memo(({ user, isUserProfilePicConfigEnabledForTenant }) =>
 
   return (
     <Row>
-      <Col xs={isUserProfilePicConfigEnabledForTenant ? 9 : 12}>
-        <div className={`${css.section} ${css.active}`}>
-          <KeyValue
-            label={<FormattedMessage id="ui-reading-room.borrower" />}
-            value={getUserValue()}
-          />
-        </div>
+      <Col xs={isUserProfilePicConfigEnabledForTenant ? 8 : 10} className={css.borrowerDetails}>
+        <KeyValue
+          label={<FormattedMessage id="ui-reading-room.borrower" />}
+          value={getUserValue()}
+        />
```

Comment:
Could you please use the same code style approach?
For `Col` you add both props within one line, but for `KeyValue` you use two separate lines.